staging/src/k8s.io/apiserver/pkg/server/filters/reqmgmt.go

```diff
@@ -80,6 +81,8 @@ func WithRequestManagement(
 			}
 			afterExecute()
 		} else {
+			klog.V(6).Infof("Rejecting RequestInfo=%#+v, user.Info=%#+v\n", requestInfo, user)
```

Comment:
#81679 removed this log statement because it was redundant with and less informative than the ones added.  In the currently merged code, as updated by #81679 , every path through `reqMgmt.Wait` has a log message (and I think it should stay this way).

Reply:
fixed